.github/workflows/main.yml

```diff
@@ -40,7 +40,7 @@ jobs:
 
     - name: 🔨 Build & Test
       run: |
-        stack --no-terminal test --bench --haddock --no-haddock-deps --no-run-benchmarks --flag cardano-addresses:release --flag cardano-addresses-cli:release
+        stack --no-terminal test --no-run-tests --bench --haddock --no-haddock-deps --no-run-benchmarks --flag cardano-addresses:release --flag cardano-addresses-cli:release
```

Comment:
We are already running the tests right after as part of the coverage calculation yet, we still want to build the doc and with the proper "release" flag, but no need to run the tests twice.

Reply:
ok, I was just about to ask...